### DISCUSSION TITLE
How can I deactivate V4L2 camera? Only want to use network camera in motioneye

### DISCUSSION THREAD

**Discussion by shelly20222** [Motion-Project/motion#1623] [Q&A]
The syslog on my raspberry pi is full of this messages. I am only using network cameras in motioneye. So I would like to disable V4L2 camera support.

`
Jan 24 09:54:20 raspberrypi motion: [1:ml1] [ALR] [VID] v4l2_device_open: Failed                         to open video device /dev/video0: No such file or directory

Jan 24 09:54:20 raspberrypi motion: [1:ml1] [ERR] [VID] vid_start: V4L2 device f                        ailed to open

Jan 24 09:54:30 raspberrypi motion: [1:ml1] [WRN] [ALL] mlp_retry: Retrying until successful connection with camera

Jan 24 09:54:30 raspberrypi motion: [1:ml1] [NTC] [VID] vid_start: Opening V4L2 device

Jan 24 09:54:30 raspberrypi motion: [1:ml1] [NTC] [VID] v4l2_device_open: Using videodevice /dev/video0 and input -1

Jan 24 09:54:30 raspberrypi motion: [1:ml1] [ALR] [VID] v4l2_device_open: Failed to open video device /dev/video0: No such file or directory

Jan 24 09:54:30 raspberrypi motion: [1:ml1] [ERR] [VID] vid_start: V4L2 device failed to open
`

**Comment by shelly20222** [comment]
Has someone an answer to this? I am still struggeling

**Reply by tosiara** [reply]
try to run motion standalone, reproduce the issue, upload the full log and config
